packages/@aws-cdk/aws-redshift-alpha/lib/table.ts

```diff
@@ -253,6 +261,7 @@ export class Table extends TableBase {
     this.tableColumns = this.configureTableColumns(props.tableColumns);
     this.cluster = props.cluster;
     this.databaseName = props.databaseName;
+    this.id = id;
```

Comment:
wait... this is just the construct id right? You should be able to reference this within the construct already with `this.node.id`. I don't think we need to specify an additional property for it

Reply:
Made my work simpler lol